src/components/AccountDetailsV2/index.tsx

```diff
@@ -17,13 +17,14 @@ export enum TransactionState {
   Failed,
 }
 
-const Grid = styled.a`
+const Grid = styled.a<{ isLastTransactionInList?: boolean }>`
```

Comment:
consider using `src/theme/components`'s `ExternalLink` i.e.,

```suggestion
const Grid = styled(ExternalLink)<{ isLastTransactionInList?: boolean }>`
```

Reply:
ty done